operation/src/main/scala/io/vamp/operation/gateway/RouteComparator.scala

```diff
@@ -57,13 +58,13 @@ trait RouteComparator extends LazyLogging {
         (currentRoute.conditionStrength, nextRoute.conditionStrength) match {
           case (Some(currentConditionStrength), Some(nextConditionStrength)) ⇒
             if (currentConditionStrength.value != nextConditionStrength.value)
-              sendRouteEvent(gateway, "conditionstrengthupdated", key, caller)
+              sendRouteEvent(updatedGateway, "conditionstrengthupdated", key, caller)
             else
               logDebug(s"RouteEvents Condition Strength didn't change for key: $key $caller")
           case (None, Some(_)) ⇒
-            sendRouteEvent(gateway, "conditionstrengthnadded", key, caller)
+            sendRouteEvent(updatedGateway, "conditionstrengthnadded", key, caller)
```

Comment:
Can we fix this typo? conditionstrengthnadded -> conditionstrengthadded

Reply:
Sure, I'll fix that